src/test/scala/firrtlTests/CustomTransformSpec.scala

```diff
@@ -79,7 +79,6 @@ class CustomTransformSpec extends FirrtlFlatSpec {
       def inputForm = HighForm
       def outputForm = HighForm
       def execute(s: CircuitState) = {
-        println(name)
```

Comment:
Removal of this actually negates what this test is checking. Namely, this is asserting that calling the `name` method doesn't cause a crach. I am in favor of keeping the verbosity down here, so maybe this could be changed to a bare `name` call? (see: https://github.com/freechipsproject/firrtl/pull/1228)

Reply:
Your solution is better than my `println`. Thanks!